tests/secure-supply-chain-tests.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+source $BATS_TEST_DIRNAME/common.bash
+
+SECURE_SUPPLY_CHAIN_VERIFICATION_CONFIG_MAP_NAME="ssc-verification-config"
+
+teardown_file() {
+	helm upgrade --set policyServer.verificationConfig="" --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defaults
+	wait_for_default_policy_server_rollout
+	kubectl_delete_configmap_by_name $SECURE_SUPPLY_CHAIN_VERIFICATION_CONFIG_MAP_NAME
+}
+
+setup() {
+	clean_up_environment
+	kubectl_delete_configmap_by_name $SECURE_SUPPLY_CHAIN_VERIFICATION_CONFIG_MAP_NAME
+}
+
+@test "[Secure Supply Chain tests] Trusted policy should not block policy server" {
+	create_configmap_from_file_with_root_key $SECURE_SUPPLY_CHAIN_VERIFICATION_CONFIG_MAP_NAME verification-config $RESOURCES_DIR/secure-supply-chain-verification-config.yaml
+	helm upgrade --set policyServer.verificationConfig=$SECURE_SUPPLY_CHAIN_VERIFICATION_CONFIG_MAP_NAME --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defaults
+	kubectl_apply_should_succeed $RESOURCES_DIR/privileged-pod-policy.yaml
+	wait_for_all_cluster_admission_policies_to_be_active
+	wait_for_default_policy_server_rollout
```

Comment:
I'm aware that we have a [function](https://github.com/kubewarden/kubewarden-end-to-end-tests/blob/main/tests/common.bash#L23) calling similar functions. But in my tests, the function is not working properly and I need to double check why. I believe is an issue in the function used to check policy condition. I'll investigate that and update this PR or open another one to fix the issue.